.yamato/upm-ci-renderstreaming-packages.yml

```diff
@@ -238,9 +238,9 @@ publish_dry_run_{{ package.name }}:
         - "upm-ci~/packages/*.tgz"
   dependencies:
     - .yamato/upm-ci-renderstreaming-packages.yml#pack_{{ package.name }}
-    {% for editor in editors %}
-    - .yamato/upm-ci-renderstreaming-packages.yml#trigger_package_test_{{ package.name }}_{{ editor.version }}
-    {% endfor %}
+    - .yamato/upm-ci-renderstreaming-packages.yml#test_{{ package.name }}_editmode_mono_win_2019.4
+    - .yamato/upm-ci-renderstreaming-packages.yml#test_{{ package.name }}_editmode_mono_macos_2019.4
+    - .yamato/upm-ci-renderstreaming-packages.yml#test_{{ package.name }}_editmode_mono_linux_2019.4
```

Comment:
I think this depends only on the `editmode` test, is that correct?
I think needs to depend other test (playmode , standalone...)

Reply:
It is correct. The test with dry-run option is failed when depending playmode or standalone test.
As you can see this setting in the WebRTC package.
https://github.com/Unity-Technologies/com.unity.webrtc/blob/develop/.yamato/upm-ci-webrtc-packages.yml#L450-L472